app/__init__.py

```diff
@@ -55,7 +56,9 @@ def home():
     @app.route('/admin')
     @login_required
     def admin():
-        return render_template('admin.html')
+        users = get_all_users()
+        #print(users)
+        return render_template('admin.html',users=users)
```

Comment:
Passing-in the current "user" into the route handler will help the Flask component render the "Admin" in the menu option  
```        
user = user = db.child('users').child(g.user['localId']).get().val()
return render_template('admin.html',users=users, user=user)
```
@hlopezes14